pages/linux/gcov.md

```diff
@@ -0,0 +1,29 @@
+# gcov
+
+> Code coverage analysis and profiling tool. Discover untested parts of your program.
+> Get copy of source code annotated with execution frequencies of code segments.
+> More information: <https://gcc.gnu.org/onlinedocs/gcc/Invoking-Gcov.html>.
+
+- Generate coverage report named `file.cpp.gcov`:
+
+`gcov {{path/to/file.cpp}}`
+
+- Write individual execution counts for every basic block:
+
+`gcov --all-blocks {{path/to/file.cpp}}`
+
+- Write branch frequencies to output file and print summary information to `stdout` as a percentage:
```

Comment:
Perhaps this will read better?
```suggestion
- Write branch frequencies to the output file and print summary information to `stdout` as a percentage:
```